apps/dashboard/src/components/workflow-editor/steps/email/email-tabs.tsx

```diff
@@ -58,7 +58,7 @@ export const EmailTabs = (props: StepEditorProps) => {
       <Separator />
       <TabsContent value="editor" forceMount className={tabsContentClassName}>
         {workflow.origin === WorkflowOriginEnum.NOVU_CLOUD && uiSchema && <EmailEditor uiSchema={uiSchema} />}
-        {workflow.origin === WorkflowOriginEnum.EXTERNAL && (
+        {workflow.origin === WorkflowOriginEnum.EXTERNAL && !uiSchema && (
```

Comment:
@BiswaViraj A short-tem goal is to unify uiSchema across all control types. That is, use it for custom step controls as well. That is, I suggest removing this check from the UI.